os/net/lwip/configs/tcp/Kconfig

```diff
@@ -18,6 +18,23 @@ config NET_TCP_WND
 		The size of a TCP window.  This must be at least (2 * TCP_MSS)
 		for things to work well
 
+config NET_WND_SCALE
+	bool "Window scale support"
+	default n
+	---help---
+		Set LWIP_WND_SCALE to 1 to enable window scaling.
+
+
+if NET_WND_SCALE
+config NET_TCP_RCV_SCALE
+	int "Receive window scale support"
+	default 0
+	---help---
+		Set TCP_RCV_SCALE to the desired scaling factor (shift count in the range of [0..14]).
```

Comment:
If this config has value range, how about adding `range a b`?
Then we can prevent invalid value.

Reply:
Thanks. It's better to reduce mistake.